### PR TITLE
Fixes #4: Use robo packages as task runners.

### LINKED ISSUE #4: Use robo packages as task runners

### PR DESCRIPTION
**Pull request by Jelle-S**
Fixes #4

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    {
+        $currentProjectRoot = $remote['currentdir'] . '/..';
+        $collection = $this->collectionBuilder();
+        $parent = parent::postSymlinkTask($worker, $auth, $remote);
+        if ($parent) {
+            $collection->addTask($parent);
+        }
+        $collection->taskSsh($worker, $auth)
+            ->remoteDirectory($currentProjectRoot, true)
+            ->exec('composer symfony-scripts');
```
Comment (daften) [inline RoboFile.php]: Isn't the symlink executed before composer install? Or is it the other way round and is that why this is removed from post-install? In which case, shouldn't we adapt the general symfony flow?

Comment (Jelle-S) [inline RoboFile.php]: Composer install is executed on the build server, the symlinks on the test/qa/prod servers. problem is dat the symfony scripts load config, which loads parameters.yml, which isn't available until after the symlinks.

Comment (daften) [inline RoboFile.php]: And composer install --no-scripts on the build server, is that an option? :)

Comment (Jelle-S) [inline RoboFile.php]: Possibly (probably), until we have a symfony project that uses patches in the composer file (like in our Drupal projects), and this code would still need to be here (the symfony-scripts build the assets and whatnot)

**Pull request merged**